v1/client_test.go

```diff
@@ -30,18 +37,32 @@ func ExampleNewClient() {
 	log.Println(client)
 }
 
-func TestWriteSchemaCall(t *testing.T) {
-	t.Parallel()
-	require := require.New(t)
+func randomString(length int) (string, error) {
+	buffer := make([]byte, length)
+	_, err := rand.Read(buffer)
+	if err != nil {
+		return "", err
+	}
+	return base64.StdEncoding.EncodeToString(buffer)[:length], nil
+}
 
-	// TODO: should we get a handle on the connection in order to be able to close it?
-	// It should only matter in testing, but it could still be a problem.
+func testClient(t *testing.T) *authzed.Client {
+	token, err := randomString(12)
+	require.NoError(t, err)
 	client, err := authzed.NewClient(
 		"localhost:50051",
 		grpc.WithTransportCredentials(insecure.NewCredentials()),
-		grpcutil.WithInsecureBearerToken("some token"),
+		grpcutil.WithInsecureBearerToken(token),
 	)
-	require.NoError(err)
+	require.NoError(t, err)
```

Comment:
you can call `t.Cleanup(client.conn.Close)` here and then you don't have to `defer client.conn.Close()` in every test